src/views/forms/TradeForm/AmountCloseInput.tsx

```diff
@@ -0,0 +1,117 @@
+import { NumberFormatValues } from 'react-number-format';
+import styled from 'styled-components';
+
+import { useQuickUpdatingState } from '@/hooks/useQuickUpdatingState';
+
+import breakpoints from '@/styles/breakpoints';
+import { formMixins } from '@/styles/formMixins';
+
+import { Input, InputType } from '@/components/Input';
+import { Slider } from '@/components/Slider';
+import { WithLabel } from '@/components/WithLabel';
+
+import { mapIfPresent } from '@/lib/do';
+import { MustBigNumber } from '@/lib/numbers';
+
+export const AmountCloseInput = ({
+  amountClosePercentInput,
+  setAmountCloseInput,
+}: {
+  amountClosePercentInput: string | undefined;
+  setAmountCloseInput: (val: string | undefined) => void;
+}) => {
+  const {
+    value: amountClose,
+    setValue: setAmountClose,
+    commitValue: commitAmountClose,
+  } = useQuickUpdatingState<string | undefined>({
+    setValueSlow: setAmountCloseInput,
+    slowValue: amountClosePercentInput,
+    debounceMs: 100,
+  });
+
+  const onSliderDrag = ([newValue]: number[]) => {
+    const newValueString = mapIfPresent(newValue, (lev) => MustBigNumber(lev).toFixed(0));
+    setAmountClose(newValueString ?? '');
+  };
+
+  const commitValue = (newValue: string | undefined) => {
+    commitAmountClose(newValue);
+  };
+
+  const onValueCommit = ([newValue]: number[]) => {
+    commitValue(MustBigNumber(newValue).toFixed(0));
+  };
+
+  return (
+    <$InputContainer>
+      {/* TODO: localize */}
+      <$WithLabel label={<div tw="flex">Amount Close</div>}>
+        <$AmountCloseSlider
+          label="AmountClose"
+          min={0}
+          max={100}
+          step={0.1}
+          value={MustBigNumber(amountClose).toNumber()}
+          onSliderDrag={onSliderDrag}
+          onValueCommit={onValueCommit}
+        />
+      </$WithLabel>
+      <$InnerInputContainer>
+        <Input
+          placeholder={`${MustBigNumber(amountClose).toFixed(0)}%`}
+          type={InputType.Percent}
+          value={amountClosePercentInput}
+          max={100}
+          onChange={({ value }: NumberFormatValues) => {
+            commitValue(value);
+          }}
+        />
+      </$InnerInputContainer>
+    </$InputContainer>
+  );
+};
+
+const $InputContainer = styled.div`
+  ${formMixins.inputContainer}
+  --input-height: 3.5rem;
+  --input-backgroundColor: none;
```

Comment:
Are we moving away from having input color? I noticed this for Market leverage as well, but the TargetLeverage component still has a bg color.